assets/schema_input.json

```diff
@@ -43,78 +46,78 @@
             "bowtie1_index": {
                 "meta": ["bowtie1_index"],
                 "type": "string",
-                "errorMessage": "TODO"
+                "errorMessage": "Bowtie1 index"
```

Comment:
no pattern? How about unifying the errorMessages? `, cannot contain spaces`

Reply:
it's a folder, so no pattern, but I can add a pattern for no spaces indeed